Assets/MixedRealityToolkit/Providers/BaseCameraSettingsProvider.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Microsoft.MixedReality.Toolkit.CameraSystem
 {
-    public class BaseCameraSettingsProvider : BaseDataProvider, IMixedRealityCameraSettingsProvider
+    public abstract class BaseCameraSettingsProvider : BaseDataProvider, IMixedRealityCameraSettingsProvider
```

Comment:
Along the lines of https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6564 and https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1012?view=vs-2019, it might be good to also update the constructors to `protected`.

Reply:
good catch. i'll make that change